sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -385,6 +385,10 @@ multilib_src_install_all() {
 	systemd_enable_service multi-user.target systemd-resolved.service
 	systemd_enable_service sysinit.target systemd-timesyncd.service
 
+	# enable getty manually
+	mkdir --parents "${D}/usr/lib/systemd/system/getty.target.wants"
```

Comment:
In the lines below and above `ED` is used instead of `D`; I would stick to it to include the `EPREFIX` in case this is needed.